transforms/__tests__/rename-unsafe-lifecycles-test.js

```diff
@@ -30,12 +24,53 @@ const tests = [
 const defineTest = require('jscodeshift/dist/testUtils').defineTest;
 
 describe('rename-unsafe-lifecycles', () => {
-  tests.forEach(test =>
+  describe('flow', () => {
+    beforeEach(() => {
+      jest.mock('../rename-unsafe-lifecycles', () => {
+        return Object.assign(
+          require.requireActual('../rename-unsafe-lifecycles'),
+          {
+            parser: 'flow'
+          }
+        );
+      });
+    });
+
+    afterEach(() => {
+      jest.resetModules();
```

Comment:
This was necessary to clear the `jest.mock` above, otherwise the following `typescript` block was ran with the `flow` parser because of the above mock. [Jest Docs](https://jestjs.io/docs/en/jest-object.html#jestresetmodules).

Thought about doing a `beforeAll`/`afterAll` since all the flow tests need the same mock, or promoting this `afterEach` to the top-level `describe`. Thought this was cleaner to run before/after every test and keep it specific to where it's needed but happy to change.